javawrapper/device/iothub-client-javawrapper/DeviceClient.java

```diff
@@ -0,0 +1,130 @@
+// Copyright(c) Microsoft.All rights reserved. 
```

Comment:
Let's follow the c-api instead for this. This will simplify the binding, get rid of converting error codes to exceptions and  remove un-implemented functions (like Open() & Close())